deploy/ansible/roles/icarodb/files/icaro.sql

```diff
@@ -203,6 +203,46 @@ CREATE TABLE `session_histories` (
   `session_key` varchar(200),
   PRIMARY KEY(`id`)
 );
+
+CREATE TABLE subscription_plans (
+    id serial not null primary key,
+    code varchar(1024) not null,
+    name varchar(1024) not null,
+    description character varchar(1024) not null,
+    price numeric,
+    period numeric default null
+);
+
+INSERT INTO subscription_plans VALUES (1, 'free', 'Free', 'Free limited plan', 0.00, 365);
+INSERT INTO subscription_plans VALUES (2, 'basic', 'Basic', 'Basic plan', 0.00, 365);
+INSERT INTO subscription_plans VALUES (3, 'standard', 'Standard', 'Standard lan', 0.00, 365);
+INSERT INTO subscription_plans VALUES (4, 'premium', 'Premium', 'Premium plan', 0.00, 365);
+
+CREATE TABLE subscriptions (
+    id serial not null primary key,
+    user_id varchar(1024) not null,
```

Comment:
maybe account_id is better because it's a reseller and not a user

Reply:
Agree, also account_id should be bigint and a reference to accounts table.